.github/workflows/pylint.yml

```diff
@@ -78,7 +78,7 @@ jobs:
           export PYTHONPATH=`grass --config python_path`:$PYTHONPATH
           export LD_LIBRARY_PATH=$HOME/install/grass83/lib:$LD_LIBRARY_PATH
           cd gui/wxpython
-          pylint --persistent=no --py-version=${{ matrix.min-python-version }} --jobs=$(nproc) *
+          pylint --persistent=no --py-version=${{ matrix.min-python-version }} --jobs=$(nproc) --disable=CMakeLists.txt *
```

Comment:
... this doesn't seem to help to the the related CI step passing ok. Ideas anyone?

Reply:
(fixed in 3eae7922903f7a90cec03ea87dce1f91c1b02eff)